speech/recognize.js

```diff
@@ -162,6 +165,16 @@ function asyncRecognize (filename, encoding, sampleRateHertz, languageCode) {
     .then((results) => {
```

Comment:
I would rename this variable to something different (e.g response) to avoid the confusing naming, both here and in the other examples.